Xamarin.Forms.Platform.Android/Renderers/ScrollViewRenderer.cs

```diff
@@ -90,7 +92,7 @@ public void SetElement(VisualElement element)
 				if (_container == null)
 				{
 					Tracker = new VisualElementTracker(this);
-					_container = new ScrollViewContainer(_view, Context);
+					_container = new ScrollViewContainer(_view, _context);
```

Comment:
@jsuarezruiz I think this is the change that fixed #9103, so let's try to keep this if we can.

Reply:
I have changed this by applying changes similar to those applied by E.Z in #10893.